industry_benchmarks/utils/data_gathering.py

```diff
@@ -510,6 +793,10 @@ def gather_data(
     with open(file, mode='w') as f:
         json.dump(network_properties, f)
 
+    # finally zip the folder
+    shutil.make_archive("all_results.zip", "zip", output_dir.as_posix())
```

Comment:
We need to document what's in the archive into the JSON. This is so that our partners have a human readable-ish account of everything they are shipping.